packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -257,11 +245,11 @@ export default function ViewApplicationPage() {
       setAnswerBlocks(_answerBlocks);
     };
 
-    if (isApplicationFetched && hasAccess) {
+    if (isLoading && hasAccess) {
```

Comment:
this should probably be `application && !isLoading` (`isSuccess`?) - since the application needs to be fetched before the answers can be decrypted (next line)

Reply:
fixeddd